deluge/core/rpcserver.py

```diff
@@ -374,6 +374,7 @@ def __init__(self, port=58846, interface="", allow_remote=False, listen=True):
             reactor.listenSSL(port, self.factory, ServerContextFactory(), interface=hostname)
         except Exception as ex:
             log.info("Daemon already running or port not available..")
+            log.error("Error when listening on %s:%s", hostname, port)
```

Comment:
This could be combined with the log.error below. 

`"Error attempting to listen on %s:%s: %s", hostname, port, ex`